python_modules/dagster/dagster/_core/execution/context_creation_pipeline.py

```diff
@@ -63,7 +63,7 @@ def initialize_console_manager(
 ) -> DagsterLogManager:
```

Comment:
@alangenfeld this is the sticking point for your 'can it be made non-optional' question, there's a long tail of callsites for this off the beaten path where there is no instance currently

Reply:
gotcha - maybe a test to ensure its respected in `execute_job` / `execute_in_process` then to protect over time